db/re/pet_evolve_db.conf

```diff
@@ -0,0 +1,254 @@
+//================= Hercules Database =====================================
+//=       _   _                     _
+//=      | | | |                   | |
+//=      | |_| | ___ _ __ ___ _   _| | ___  ___
+//=      |  _  |/ _ \ '__/ __| | | | |/ _ \/ __|
+//=      | | | |  __/ | | (__| |_| | |  __/\__ \
+//=      \_| |_/\___|_|  \___|\__,_|_|\___||___/
+//================= License ===============================================
+//= This file is part of Hercules.
+//= http://herc.ws - http://github.com/HerculesWS/Hercules
+//=
+//= Copyright (C) 2018  Hercules Dev Team
+//= Copyright (C) 2018 Dastgir
+//=
+//= Hercules is free software: you can redistribute it and/or modify
+//= it under the terms of the GNU General Public License as published by
+//= the Free Software Foundation, either version 3 of the License, or
+//= (at your option) any later version.
+//=
+//= This program is distributed in the hope that it will be useful,
+//= but WITHOUT ANY WARRANTY; without even the implied warranty of
+//= MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//= GNU General Public License for more details.
+//=
+//= You should have received a copy of the GNU General Public License
+//= along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//=========================================================================
+//= Pet Evolution Database
+//=========================================================================
+
+pet_evolve: (
+/**************************************************************************
+ ************* Entry structure ********************************************
+ **************************************************************************
+{
+	// ================ Mandatory fields ==============================
+	Id: ID                               (int, Evolved Pet EggID)
+	From: ID                             (int, Evolving Pet EggID)
+	Items: {
+		Name: Amount                     (items required to perform evolution)
+		...
+	}
+},
+**************************************************************************/
+{
+	Id: 9069
```

Comment:
Would not it be better to use aegis name instead of id? Constant makes it easy to know which monster will evolve.

Reply:
probably better merge this db with petdb.
and use names in all fields and not id.

Reply:
@4144 Done